client/src/screens/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { BrowserRouter, Route, Routes as Switch } from "react-router-dom";
+
+import Login from "./Login/Login";
+import Register from "./Register/Register";
+import Profile from "./Profile/Profile";
+import EditDog from "./EditDog/EditDog";
+import AddDog from "./AddDog/AddDog";
+import DogsList from "./DogsList/DogsList";
+
+export default function Router() {
+  return (
+    <BrowserRouter>
+      <Switch>
+        <Route path="/login" element={<Login />} />
+        <Route path="/register" element={<Register />} />
+        <Route path="/users/:userId" element={<Profile />} />
+        <Route path="/dogs/new" element={<AddDog />} />
```

Comment:
remove this path , it's include in `/dogs/:dogId` path

Reply:
removed